akka-stream/src/main/scala/akka/stream/SslTlsOptions.scala

```diff
@@ -0,0 +1,255 @@
+/**
+ * Copyright (C) 2015-2016 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream
```

Comment:
Still inconsistent. Remember that the API is going to be used way more than it is written, so saving busywork for a few but creating inconsistencies for many doesn't seem like a great tradeoff IMO.